File: keycloak/26.7.1/analyzed/js/apps/admin-ui/src/context/auth/uiRealmInfo.ts
/**
 * 领域级 UI 元信息：对该领域所有管理员可见的运行时开关与能力描述。
 */
export interface UiRealmInfo {
  /** 是否至少启用了一个用户存储/用户配置 Provider */
  userProfileProvidersEnabled?: boolean;
}

File: keycloak/26.7.1/analyzed/rest/admin-ui-ext/src/main/java/org/keycloak/admin/ui/rest/model/UIRealmInfo.java
package org.keycloak.admin.ui.rest.model;

/**
 * 领域信息摘要，对该领域的每位管理员可见，不限于有权查看完整领域配置的管理员。
 * <p>
 * 用于在管理 UI 中展示与当前登录管理员相关的领域级能力开关等信息。
 */
public class UIRealmInfo {
    /** 是否启用了用户配置文件（User Profile）相关提供者。 */
    private boolean userProfileProvidersEnabled;

    public boolean isUserProfileProvidersEnabled() {
        return userProfileProvidersEnabled;
    }

    public void setUserProfileProvidersEnabled(final boolean userProfileProvidersEnabled) {
        this.userProfileProvidersEnabled = userProfileProvidersEnabled;
    }
}
